commerce_bluesnap.module

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Provides Commerce integration with the BlueSnap Payment Gateway.
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_help().
+ */
+function commerce_bluesnap_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    case 'help.page.commerce_bluesnap':
+      $output = '';
+      $output .= '<h2>' . t('About') . '</h2>';
+      $output .= '<p>' . t('Provides Drupal Commerce integration with the BlueSnap Payment Gateway.') . '</p>';
+      $output .= '<h3>' . t('Features') . '</h3>';
+      $output .= '<ul>';
+      $output .= '<li>' . t(
+          '<strong>Credit Card Payments Integration</strong>
+          - Hosted Payment Fields payment gateway. Serves payment input fields
+          from Bluesnap’s domain, minimizing the scope of PCI compliance.
+          <br> For more info, see the README.txt file.
+          Configure <a href=":configuration_link">here</a>.', [
+            ':configuration_link' => Url::fromUri(
+              'internal:/admin/commerce/config/payment-gateways/manage/bluesnap'
+            )->toString()
+          ]
+        ) .
+        '</li>';
+      $output .= '</ul>';
+      return $output;
+  }
+}
```

Comment:
Dimitris, actually Drupal recommends doing it this way (see install_check_translations()):

![image](https://user-images.githubusercontent.com/11439056/56339493-285e4480-61cc-11e9-8d71-83e79592ee9c.png)

I did try it the way you recommended by using the html tags in the parameters, but it looks like this:

![image](https://user-images.githubusercontent.com/11439056/56339405-cd2c5200-61cb-11e9-9382-f1980d8fcc95.png)

The method that we currently have displays it fine:

![image](https://user-images.githubusercontent.com/11439056/56339571-66f3ff00-61cc-11e9-9145-1615666004d4.png)

Reply:
@shabananavas Good to know that HTML tags are supported within `t()`, it would be good to validate at some point that whitespaces are ok too as the example from Drupal Core do not break the string to multiple lines while we want to do that for better reliability.